### ISSUE TITLE
Neocomplete stopped working due to missing Lua support on Arch Linux

### ISSUE THREAD

**Issue by lervag** [Shougo/neocomplete.vim#572]
This is not really an issue with neocomplete, but I thought it might be of interest to more neocomplete users and so I post it anyway. After a recent update on Arch Linux, I now get the following message:

```
neocomplete requires Vim 7.3.885 or later with Lua support ("+lua").
```

My Vim version is 8.0.237 and it has Lua support. However, it seems there is now a bug in the dynamic linking. It works if I `set luadll=liblua.so`, instead of the default `luadll=liblua.so.5.3.3`. Note that the `liblua.so.5.3.3` file _does_ exist, but for some reason it does not seem to work.

I expect this issue will be fixed in not too long from the Arch repos. If you should have any further understanding of the issue, I would be happy to hear it. Else I would propose that the issue remains open for some time in case others experience the same problem.

**Comment by Shougo** [comment]
Hm.

**Comment by pojoin** [comment]
I have the same problem.

**Comment by Shougo** [comment]
It is Arch Linux issue.
You should report it to Arch maintainer.

**Comment by lervag** [comment]
It seems to be fixed in updated version 8.0.314 that's been pushed to Arch repos now.
